src/software/firmware/srcs/qualification.cpp

```diff
@@ -477,22 +548,22 @@ void loop() {
     }
     case STEP_LED_RED: {
         UNGREEDY(is_drawn, display("Red LED is ON", "Press start"));
-        digitalWrite(PIN_LED_RED, HIGH);
+        digitalWrite(PIN_LED_RED, LED_RED_ACTIVE);
```

Comment:
I don't find the same modification for LEDs in `alarm_controller.cpp`. 
Do you want to manage this in a second PR?

Reply:
Done!